2.0/contrib/src/.bashrc

```diff
@@ -0,0 +1,2 @@
+# This will make scl collection binaries work out of box.
+source scl_source enable ruby200 ror40
```

Comment:
It is base on [this line](https://github.com/openshift/sti-base/blob/master/Dockerfile#L67)